app/Models/EntryDAOSQLite.php

```diff
@@ -114,13 +114,13 @@ protected function updateCacheUnreads($catId = false, $feedId = false) {
 	public function markRead($ids, $is_read = true) {
 		FreshRSS_UserDAO::touch();
 		if (is_array($ids)) {	//Many IDs at once (used by API)
-			if (true) {	//Speed heuristics	//TODO: Not implemented yet for SQLite (so always call IDs one by one)
+			//if (true) {	//Speed heuristics	//TODO: Not implemented yet for SQLite (so always call IDs one by one)
```

Comment:
I'd rather remove it completely than having some commented code.

Reply:
Yes indeed, but in this specific case, it is to match the construct of the parent code and make the TODO easier to understand & address:
https://github.com/FreshRSS/FreshRSS/blob/064288ed6228d3d56b2231aadd507eb62ae8662c/app/Models/EntryDAO.php#L338